pom.xml

```diff
@@ -100,6 +106,29 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>com.diffplug.spotless</groupId>
+				<artifactId>spotless-maven-plugin</artifactId>
+				<version>${spotless-maven-plugin.version}</version>
+				<configuration>
+					<java>
+						<importOrder />
+						<removeUnusedImports />
+						<palantirJavaFormat>
+							<version>${spotless-maven-plugin.version}</version>
```

Comment:
Thanks for pointing this out! I'll create a separate property for the `palantir-java-format` version to ensure it can be managed independently from the `spotless-maven-plugin.version`.